### ISSUE TITLE
repeating 404 for `GET /apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations`

### ISSUE THREAD

**Issue by MikeSpreitzer** [kubernetes/minikube#3598]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

Please provide the following details:

**Environment**: Minikube

**Minikube version** (use `minikube version`): v0.33.1
- **OS** (e.g. from /etc/os-release): MacOS 10.13.6
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): virtualbox
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): v0.33.1
- **Install tools**: I used homebrew to install minikube
- **Others**:
```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.2", GitCommit:"cff46ab41ff0bb44d8584413b598ad8360ec1def", GitTreeState:"clean", BuildDate:"2019-01-13T23:16:58Z", GoVersion:"go1.11.4", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.2", GitCommit:"cff46ab41ff0bb44d8584413b598ad8360ec1def", GitTreeState:"clean", BuildDate:"2019-01-10T23:28:14Z", GoVersion:"go1.11.4", Compiler:"gc", Platform:"linux/amd64"}
```

**What happened**:
I started fresh (`minikube delete`; `rm -rf ~/.minikube`), with -v=5 logging for the kube apiserver.  Minikube came up, and I saw repeating 404s for GETs of initializerconfigurations.

```
$ minikube start --extra-config=apiserver.v=5
Starting local Kubernetes v1.13.2 cluster...
Starting VM...
Downloading Minikube ISO
 181.48 MB / 181.48 MB [============================================] 100.00% 0s
Getting VM IP address...
Moving files into cluster...
Downloading kubeadm v1.13.2
Downloading kubelet v1.13.2
Finished Downloading kubeadm v1.13.2
Finished Downloading kubelet v1.13.2
Setting up certs...
Connecting to cluster...
Setting up kubeconfig...
Stopping extra container runtimes...
Starting cluster components...
Verifying kubelet health ...
Verifying apiserver health ...
Kubectl is now configured to use the cluster.
Loading cached images from config file.


Everything looks great. Please enjoy minikube!
```

Here is an excerpt from the kube apiserver log:
```
I0128 23:59:41.301852       1 handler.go:153] kube-aggregator: GET "/apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations" satisfied by nonGoRestful
I0128 23:59:41.302037       1 pathrecorder.go:253] kube-aggregator: "/apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations" satisfied by NotFoundHandler
I0128 23:59:41.302301       1 handler.go:143] kube-apiserver: GET "/apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations" satisfied by gorestful with webservice /apis/admissionregistration.k8s.io
I0128 23:59:41.302524       1 wrap.go:47] GET /apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations: (821.242µs) 404 [kube-apiserver/v1.13.2 (linux/amd64) kubernetes/cff46ab [::1]:46878]
I0128 23:59:41.302984       1 initializer_manager.go:45] Initializers are disabled due to an error: the server could not find the requested resource
I0128 23:59:41.346731       1 handler.go:153] kube-aggregator: GET "/apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations" satisfied by nonGoRestful
I0128 23:59:41.346921       1 pathrecorder.go:253] kube-aggregator: "/apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations" satisfied by NotFoundHandler
I0128 23:59:41.347002       1 handler.go:143] kube-apiserver: GET "/apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations" satisfied by gorestful with webservice /apis/admissionregistration.k8s.io
I0128 23:59:41.347272       1 wrap.go:47] GET /apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations: (617.775µs) 404 [kube-apiserver/v1.13.2 (linux/amd64) kubernetes/cff46ab [::1]:46878]
I0128 23:59:41.347750       1 initializer_manager.go:45] Initializers are disabled due to an error: the server could not find the requested resource
I0128 23:59:41.380218       1 handler.go:153] kube-aggregator: GET "/apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations" satisfied by nonGoRestful
I0128 23:59:41.380435       1 pathrecorder.go:253] kube-aggregator: "/apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations" satisfied by NotFoundHandler
I0128 23:59:41.380538       1 handler.go:143] kube-apiserver: GET "/apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations" satisfied by gorestful with webservice /apis/admissionregistration.k8s.io
I0128 23:59:41.380788       1 wrap.go:47] GET /apis/admissionregistration.k8s.io/v1alpha1/initializerconfigurations: (798.387µs) 404 [kube-apiserver/v1.13.2 (linux/amd64) kubernetes/cff46ab [::1]:46878]
I0128 23:59:41.381345       1 initializer_manager.go:45] Initializers are disabled due to an error: the server could not find the requested resource
```

I get such a batch of 3 once a second.

**What you expected to happen**:
Nothing broken out of the box.

**How to reproduce it** (as minimally and precisely as possible):
Like I described above.

**Output of `minikube logs` (if applicable)**:
(attached in a comment)

**Anything else do we need to know**:

**Comment by MikeSpreitzer** [comment]
It seems that minikube is picking up some but not all of the effects of https://github.com/kubernetes/kubernetes/pull/72972 .  I find that minikube constructs an apiserver YAML that asks for the Initializers admission controller, and if I manually remove that then the repeating 404s go away.

**Comment by tstromberg** [comment]
I believe this issue was resolved in the v1.1.0 release (more specifically, newer Kubernetes releases). Please try upgrading to the latest release of minikube, and if the same issue occurs, please re-open this bug. Thank you opening this bug report, and for your patience!

**Comment by tstromberg** [comment]
If you do run into this again, please attach `minikube logs`. With v1.1, I do not see 404's in the apiserver section. Please let me know if the logs you are referencing are somewhere other than the standard docker pod logs. Thanks!
